cmd/signing.go

```diff
@@ -119,44 +110,45 @@ func DecryptKeyShare(c *cli.Context) error {
 		)
 	}
 
-	targetFilePath := fmt.Sprintf(
-		"key_share_%.10s_%.10s",
-		keepAddress.String(),
-		keyFile.Address.String(),
-	)
-
-	if _, err := os.Stat(targetFilePath); !os.IsNotExist(err) {
-		return fmt.Errorf(
-			"could not write shares to file; file [%s] already exists",
-			targetFilePath,
-		)
-	}
-
-	err = ioutil.WriteFile(targetFilePath, signerBytes, 0444) // read-only
+	_, err = os.Stdout.Write(signerBytes)
```

Comment:
If we do support `-o`, we shouldn't write to stdout if the output file is specified.

Reply:
Addressed in the follow-up PR:  #568